boot/freeldr/freeldr/lib/fs/ext2.c

```diff
@@ -533,8 +533,6 @@ BOOLEAN Ext2ReadFileBig(PEXT2_FILE_INFO Ext2FileInfo, ULONGLONG BytesToRead, ULO
             *BytesRead += BytesToRead;
         }
         Ext2FileInfo->FilePointer += BytesToRead;
-        BytesToRead -= BytesToRead;
```

Comment:
Same comment as the one I have for fat.c

Reply:
Obviously.